pycbc/inference/models/__init__.py

```diff
@@ -183,5 +184,6 @@ def read_from_config(cp, **kwargs):
     GaussianNoise,
     MarginalizedPhaseGaussianNoise,
     MarginalizedGaussianNoise,
-    SingleTemplate
+    SingleTemplate,
+    Relative
```

Comment:
Might want to name this something like RelativeSPA or something like that just so its clear it's specific to one waveform model. Down the line, we'd like to have a more generic version, but this will likely remain.

Reply:
Okay good idea, I've renamed it as you've suggested.